purchases/src/test/java/com/revenuecat/purchases/PurchasesCommonTest.kt

```diff
@@ -1338,6 +1348,141 @@ internal class PurchasesCommonTest: BasePurchasesTest() {
         verify(exactly = 1) { mockDiagnosticsSynchronizer.syncDiagnosticsFileIfNeeded() }
     }
 
+    // endregion
```

Comment:
Something that I started doing at some point was to add the same naming in the `endregion` as the start region, so it's easier to note when things start.
```suggestion
    // endregion Diagnostics sync
```
Not a big problem though, most code isn't actually doing this 😅

Reply:
Oh, that's great to know! I'll do that, for sure. Thanks!